src/main/java/mate/academy/onlinebookstoreproject/controller/BookController.java

```diff
@@ -27,31 +28,36 @@
 public class BookController {
     private final BookService bookService;
 
+    @PreAuthorize("hasRole('ROLE_USER')")
     @GetMapping
     @Operation(summary = "Get all books", description = "Get a pageable list of all the books")
     public List<BookDto> getAll(Pageable pageable) {
         return bookService.findAll(pageable);
     }
 
+    @PreAuthorize("hasRole('ROLE_USER')")

```

Comment:
It's better to use `hasAnyRole`.
If a user with the role `ADMIN` sends request on this and the previous endpoint, he'll get status code 403